Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -53,6 +59,26 @@ for module in "${HYPERV_MODULES[@]}"; do
 done
 HYPERV_MODULES=("${tempList[@]}")
 
+if rpm -qa | grep hyper-v 2>/dev/null; then
+    expected_lis_version=$(dmesg | grep 'Vmbus LIS version' | awk -F ':' '{print $3}' | tr -d [:blank:])
+    if [[ $DISTRO_VERSION =~ 7\. ]]; then
+        HYPERV_MODULES+=('pci_hyperv')
+        pci_module=$(lsmod | grep pci_hyperv)
+        if [ -z $pci_module ]; then
+            modprobe pci_hyperv
+        fi
+    fi
+    if [[ $DISTRO_VERSION =~ 7\.3 ]] || [[ $DISTRO_VERSION =~ 7\.4 ]] ; then
+        if version_ge $expected_lis_version "4.3.0" ; then
```

Comment:
Lets use one method to compare the LIS versions. Either this or `LooseVersion(lis_version) >= LooseVersion('4.3.0')`